docs/installation.rst

```diff
@@ -14,6 +14,7 @@ Prerequisites
 * If you are new to Mozilla or the A-Team, read the `A-Team Bootcamp`_.
 * Install Git_, Virtualbox_ and Vagrant_ (latest versions recommended).
 * Clone the `treeherder repo`_ from Github.
+* Linux only: An nfsd server is required. You can install this on Ubuntu by running `apt-get install nfs-common nfs-kernel-server`
```

Comment:
This was previously mentioned here:
http://treeherder.readthedocs.io/troubleshooting.html#errors-during-vagrant-setup

However that probably isn't obvious enough.
That troubleshooting entry can be removed with the addition here.